### ISSUE TITLE
Header Top Nav clean-up

### ISSUE THREAD

**Issue by mxdvl** [guardian/dotcom-rendering#6436]
We managed to release the new header design on Friday 11th November, 2022:

- #6420
- https://github.com/guardian/frontend/pull/25659

This is currently behind a feature switch, but we need to have a few things to fix

**Short term**
- [X] Fix underlines in dropdown menu https://github.com/guardian/dotcom-rendering/pull/6425
- [X] Fix `ShowcaseLayout` https://github.com/guardian/dotcom-rendering/pull/6427
- [X] Slim header https://github.com/guardian/frontend/pull/25665
- [X] Always show the print header, whether supporter or not https://github.com/guardian/dotcom-rendering/pull/6431
- [x] Newspaper icon cut-out #6439
- [x] Peeking `SignIn` button on DCR mobile https://github.com/guardian/dotcom-rendering/pull/6434
- [x] Adjust the newspaper SVG icon to be sharper https://github.com/guardian/dotcom-rendering/issues/6436#issuecomment-1311835460

**WebX supported**
Dotcom + frontend

- [ ] Update `support-dotcom-components` with the new call to action design
- [ ] Stop `float: right` for the call to action + Logo in favour of `flex`
- [ ] Rename things i.e. `Header` & `SlimHeader`
- [x] Minimise horizontal layout shift when a user is logged in
- [ ] Turn this into the “default” header
- [ ] Reinstate skipped Cypress tests

**Design changes** ([Figma](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1255%3A171622&t=nSfg4VC2bxndeDLS-0))
- [ ] Fix slim header dropdown [e.g.](https://www.theguardian.com/uk-news/gallery/2022/nov/11/armistice-day-around-uk-in-pictures)
- [ ] Navbar hasn't changed on sports frontend pages: https://www.theguardian.com/football/world-cup-2022/overview
- [ ] Minimise layout shift when the call to action loads, with all variations of the logo
- [ ] [Dropdown on desktop and not leftCol breakpoints](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1340%3A172052&t=iIjGsRNoqGT22N6f-0) (frontend, predates new masthead navigation)
- [ ] [Hover state](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1341%3A172075&t=fBXBK0dt8s0OnV4W-0) (frontend) <--- Conversions
- [ ] [Mobile and Tablet layout issues](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1391%3A171759&t=ruZvXXyi1hXi9JeD-0) (frontend) <--- Conversions
- [ ] Revisit the amount of space to the right of the logo

**To be picked up in Q4 2023**
- [ ] [Layout issues between 980 - 1300px](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1391%3A171760&t=UvDPLbxCc0ekPdJA-0) (frontend)

**Medium term**
- [ ] Improve search behaviour
- [ ] Accessibility Audit

**Comment by mxdvl** [comment]
Hand-crafted SVG I made to relax a bit using Akemi’s beautiful set of icons as a base.

```svg
<svg width="18" height="18" viewBox="-4 -4 32 32" xmlns="http://www.w3.org/2000/svg">
    <path d="M12,-4 a15,15 0,0,0 0,32 15,15 0,0,0 0,-32
    M20 21L19 21H5L4 21V3L5 2H16L20 6V21Z
    M18 8H6V9.5H18V8Z
    M18 11H6V12.5H18V11Z
    M13 14H6V15.5H13V14Z" />
</svg>

```

**Comment by GHaberis** [comment]
I've checked and the item [Dropdown on desktop and not leftCol breakpoints](https://www.figma.com/file/h4BH9Z6d8BDi6gW3qhaA7H/single-front-door-test---header?node-id=1340%3A172052&t=iIjGsRNoqGT22N6f-0) (frontend) from the Design lists predates the introduction on the new masthead navigation, so should be one for the Web Experience platform team.

**Comment by jamesgorrie** [comment]
Closing this in favour of smaller issues.

**Comment by jamesgorrie** [comment]
This list might be useful, but not something we are trying to achieve.
